.github/workflows/python-package.yml

```diff
@@ -28,24 +28,20 @@ jobs:
       uses: actions/setup-python@v3
       with:
         python-version: ${{ matrix.python-version }}
-#        cache: 'pipenv'
     - name: Download dependencies
       run: |
         python -c "import sys; print(f'Python version: {sys.version}')"
-        python -m pip install -U setuptools pip pipenv codecov wheel
-        pipenv lock --python ${{ matrix.python-version }}
-        pipenv sync --dev
-        pipenv check || true
-        pipenv graph
+        curl -sSL https://install.python-poetry.org/ | python -
```

Comment:
You can also install poetry using pip on the line below 👍🏻

Reply:
Implemented in https://github.com/lafrenierejm/aiobotocore/commit/caf724df187d1865c1ca8d0c49672e30489267af.